win32ss/CMakeLists.txt

```diff
@@ -11,6 +11,7 @@ if(USE_DIBLIB)
 endif()
 
 add_subdirectory(gdi/gdi32)
+add_subdirectory(gdi/gdi32_vista)
```

Comment:
Would we want `gdi/gdi32/gdi32_vista`, like `dll/win32/kernel32/kernel32_vista`? (And unlike `dll/win32/advapi32_vista`..)